sdk/python/feast/permissions/auth/auth_type.py

```diff
@@ -0,0 +1,11 @@
+import enum
+
+
+class AuthType(enum.Enum):
```

Comment:
Maybe the name is now misleading as it is indicating the type of authorization server, not just the authorization type. Not sure about the best alternatives, and sorry for not noticing the change sooner.

Reply:
Maybe call it RbacType ?

Reply:
Or do you think I should break it into 2 enums again ?